src/components/ReportActionItem/TripDetailsView.tsx

```diff
@@ -0,0 +1,171 @@
+            return `${vendor}${reservation.start.location}`;
+        }
+        return reservation.start.address ?? reservation.start.location;
+    }, [reservation]);
+
+    const titleComponent = () => {
+        if (reservation.type === CONST.RESERVATION_TYPE.FLIGHT) {
+            return (
+                <View style={styles.gap1}>
+                    <View style={[styles.flexRow, styles.alignItemsCenter, styles.gap2]}>
+                        <Text style={[styles.textStrong, styles.lh20, shouldUseNarrowLayout && styles.flex1]}>{formatAirportInfo(reservation.start)}</Text>
+                        <Icon
+                            src={Expensicons.ArrowRightLong}
+                            width={variables.iconSizeSmall}
+                            height={variables.iconSizeSmall}
+                            fill={theme.icon}
+                        />
+                        <Text style={[styles.textStrong, styles.lh20, shouldUseNarrowLayout && styles.flex1]}>{formatAirportInfo(reservation.end)}</Text>
+                    </View>
+                    {bottomDescription && <Text style={[styles.textSmall, styles.colorMuted, styles.lh14]}>{bottomDescription}</Text>}
+                </View>
+            );
+        }
+
+        return (
+            <View style={styles.gap1}>
+                <Text
+                    numberOfLines={1}
+                    style={[styles.textStrong, styles.lh20]}
+                >
+                    {reservation.type === CONST.RESERVATION_TYPE.CAR ? reservation.carInfo?.name : reservation.start.longName}
+                </Text>
+                {bottomDescription && <Text style={[styles.textSmall, styles.colorMuted, styles.lh14]}>{bottomDescription}</Text>}
+            </View>
+        );
+    };
+
+    return (
+        <MenuItemWithTopDescription
+            description={formattedDate}
+            descriptionTextStyle={[styles.textLabelSupporting, styles.lh16]}
+            titleComponent={titleComponent()}
+            titleContainerStyle={[styles.justifyContentStart, styles.gap1]}
+            secondaryIcon={reservationIcon}
+            isSecondaryIconHoverable
+            shouldShowRightIcon
+            wrapperStyle={[styles.taskDescriptionMenuItem]}
+            shouldGreyOutWhenDisabled={false}
+            numberOfLinesTitle={0}
+            interactive
+            shouldStackHorizontally={false}
+            onSecondaryInteraction={() => {}}
+            iconHeight={20}
+            iconWidth={20}
+            iconStyles={[styles.tripReservationIconContainer, styles.mr3]}
+            secondaryIconFill={theme.icon}
+            hoverAndPressStyle={styles.hoveredComponentBG}
+        />
+    );
+}
+
+function TripDetailsView({tripRoomReportID, shouldShowHorizontalRule}: TripDetailsViewProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const tripTransactions = ReportUtils.getTripTransactions(tripRoomReportID);
```

Comment:
Just noticed this, but I think this will not trigger a re-render of the component if the transaction gets updated.